libcontainerd/windowsoci/oci_windows.go

```diff
@@ -66,9 +66,16 @@ type Process struct {
 	Cwd string `json:"cwd"`
 }
 
-// User contains the user information for Windows
+// User specifies user and group information for the container process.
 type User struct {
-	User string `json:"user,omitempty"`
+	// UID is the user id. (this field is platform dependent)
+	UID uint32 `json:"uid" platform:"linux,solaris"`
+	// GID is the group id. (this field is platform dependent)
+	GID uint32 `json:"gid" platform:"linux,solaris"`
+	// AdditionalGids are additional group ids set for the container's process. (this field is platform dependent)
+	AdditionalGids []uint32 `json:"additionalGids,omitempty" platform:"linux,solaris"`
+	// User user is the user name. (this field is platform dependent)
+	User string `json:"user,omitempty" platform:"windows"`
```

Comment:
Ah, it is as of this morning :smile:, but during the review it was updated to `Username`. Updated this PR accordingly. https://github.com/opencontainers/runtime-spec/pull/565